doc/source/install.rst

```diff
@@ -13,7 +27,14 @@ To install the latest development version, do
 
     pip3 install --user git+https://github.com/greenplum-db/GreenplumPython
 
+NOTE: This version is considered UNSTABLE. DON'T use it in the production environment! Stable version will be released soon.
```

Comment:
Let's remove this. There won't be another beta before GA.

Reply:
copy that